paddle/fluid/ir_adaptor/translator/op_translator.cc

```diff
@@ -2396,6 +2396,46 @@ struct RepeatInterLeaveGradOpTranscriber : public OpTranscriber {
   }
 };
 
+struct FusedElemwiseAddActivationOpTranscriber : public OpTranscriber {
+  void HandleNonexistentAttribute(pir::IrContext* ctx,
+                                  pir::AttributeMap* attribute_map,
+                                  const OpAttributeInfo& info) override {
+    if (info.name == "scale") {
+      (*attribute_map)[info.name] = pir::FloatAttribute::get(ctx, 0.0);
+    } else if (info.name == "axis") {
+      (*attribute_map)[info.name] = pir::Int32Attribute::get(ctx, -1);
+    } else if (info.name == "save_intermediate_out") {
+      (*attribute_map)[info.name] = pir::BoolAttribute::get(ctx, false);
+    }
+  }
+};
+
+struct FusedElemwiseAddActivationGradOpTranscriber : public OpTranscriber {
```

Comment:
```suggestion
struct FusedElemwiseAddActivationGradOpTranscriber : public FusedElemwiseAddActivationOpTranscriber {
```
通过继承可以复用HandleNonexistentAttribute，没必要再写一遍